core-kotlin-modules/core-kotlin-7/src/test/kotlin/typeconversion/TestCharToInt.kt

```diff
@@ -0,0 +1,38 @@
+package typeconversion
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import kotlin.test.assertNull
+
+class TestCharToInt {
+    @Test fun `Given a Char, When it's a valid Unicode character, Then should convert to the corresponding Int`() {
```

Comment:
Let's add some newlines here

```suggestion
class TestCharToInt {

    @Test
    fun `Given a Char, When it's a valid Unicode character, Then should convert to the corresponding Int`() {
```